sdk/program/src/system_instruction.rs

```diff
@@ -332,6 +371,140 @@ pub enum SystemInstruction {
     UpgradeNonceAccount,
 }
 
+/// Create an account.
+///
+/// This function produces an [`Instruction`] which must be submitted in a
+/// [`Transaction`] or [invoked] to take effect, containing a serialized
+/// [`SystemInstruction::CreateAccount`].
+///
+/// [`Transaction`]: https://docs.rs/solana-sdk/latest/solana_sdk/transaction/struct.Transaction.html
+/// [invoked]: crate::program::invoke
+///
+/// Account creation typically involves three steps: [`allocate`] space,
+/// [`transfer`] lamports for rent, [`assign`] to its owning program. The
+/// [`create_account`] function does all three at once.
+///
+/// # Required signers
+///
+/// The `from_pubkey` and `to_pubkey` signers must sign the transaction.
+///
+/// # Examples
+///
+/// These examples create a new account, allocate some space, transfer it the
+/// minimum lamports for rent exemption, and assign it to the system program.
```

Comment:
This comment seems a little strange, since the examples use the `CreateAccount` instruction. Remove this? Or rewrite to say something like "These examples use the `CreateAccount` instruction to create..."?